src/Misc/assetsManager.ts

```diff
@@ -520,6 +520,8 @@ export class TextureAssetTask extends AbstractAssetTask implements ITextureAsset
          */
         public samplingMode: number = Texture.TRILINEAR_SAMPLINGMODE) {
         super(name);
+
+        this.invertY = invertY;

```

Comment:
so why do you not let the public invertY and just set it up to false by default?

Reply:
Thanks, yes :D